mirgecom/eos.py

```diff
@@ -0,0 +1,77 @@
+__copyright__ = """
+Copyright (C) 2020 University of Illinois Board of Trustees
+"""
+
+__license__ = """
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+import numpy as np
+import numpy.linalg as la  # noqa
+from pytools.obj_array import flat_obj_array
+import pyopencl.clmath as clmath
+from meshmode.mesh import BTAG_ALL, BTAG_NONE  # noqa
+
+
+class IdealSingleGas:
+    """ Implements the ideal gas law (p = rhoRT)
```

Comment:
I don't understand?

Reply:
I was suggesting we link to [Wikipedia](https://en.wikipedia.org/wiki/Ideal_gas) here. Maybe that's not necessary.

Reply:
I am still confused.  Why would we link Wikipedia here?

Reply:
* To make sure you know how to write links in restructured text.
* To inform the more CS-leaning folks reading this of where the formula comes from and what it might mean.